### PR TITLE
docs: Fix footprint data documentation

### LINKED ISSUE #3583: Fix footprint data documentation

### PR DESCRIPTION
**Pull request by GabyCT**
This PR fixes the footprint data documentation. This replaces docker
to what we are using which is using ctr as well as it is updating
the changes that we did in the footprint script.

Fixes #3583

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: metrics/density/footprint_data.md [hunk 1]
```diff
@@ -17,21 +17,20 @@ Some variables affect how the payload is executed. The `RUNTIME` and `PAYLOAD`
...
 conditions.
 
 | Variable | Function
 | -------- | --------
-| `PAYLOAD` | The docker image to run
-| `PAYLOAD_ARGS` | Any arguments passed into the docker image
-| `PAYLOAD_RUNTIME_ARGS` | Any extra arguments passed into the docker `run` command
+| `PAYLOAD` | The ctr image to run
+| `PAYLOAD_ARGS` | Any arguments passed into the ctr image
+| `PAYLOAD_RUNTIME_ARGS` | Any extra arguments passed into the ctr `run` command
```
Comment (jodh-intel) [inline metrics/density/footprint_data.md:29]: Could you put `ctr` in backticks everywhere in this table (making this cell `ctr run`).

**Review by jodh-intel — COMMENTED** [review]
Thanks @GabyCT.

File: metrics/density/footprint_data.md (hunk 1 above)
Comment (GabyCT) [reply metrics/density/footprint_data.md]: @jodh-intel changes applied

**Review by jodh-intel — APPROVED** [review]
Thanks @GabyCT.

lgtm

**Review by devimc — APPROVED** [review]

**Pull request merged**